### DISCUSSION TITLE
Why do i owe 4 dollars to github?

### DISCUSSION THREAD

**Discussion by some-du6e** [community/community#145668] [GitHub Education] [answered]
### Hi everyone,

Does anyone know why i have to pay 4 dollars to github? I dont know why that happened because my old student benefits expire on the 15 of december. 
![Screen Shot 2024-11-26 at 11 50 08 AM](https://github.com/user-attachments/assets/34411e66-64b1-41fe-b639-d18fe5e329ca)

**Comment by CyberSphinxxx** [comment, marked as answer]
Hi there @some-du6e,

The $4 charge you’re seeing is likely a scheduled payment notification. GitHub is preparing to transition your account to a paid plan (e.g., GitHub Pro) once your student benefits expire on December 15. The payment of $4 is scheduled to be due on December 17, but it hasn’t been charged yet.

### Why This Happens:
- When student benefits expire, GitHub automatically upgrades your account to a paid plan unless you manually switch to the free plan.
- The notification is just a reminder that your billing will start after the expiration of your benefits.

### How to Avoid the $4 Charge:
1. Go to **Settings > Billing & Plans** in your GitHub account.
2. Click **Manage Plan** and switch to the **Free Plan**.
3. Do this before December 17, and you won’t be charged.

If you’ve already downgraded or believe the charge is an error, you can reach out to GitHub Support for clarification.

I hope this helps!

**Comment by olbotta** [comment]


**Reply by some-du6e** [reply]
i didnt get billed tho

**Comment by cielhaidir** [comment]
can I cancel this charge? GitHub Developer Plan – Monthly (Feb 24, 2026 – Mar 23, 2026). I haven’t even used it yet. Thankfully, the payment was declined, but now I can’t trigger GitHub Actions because of it. 😅
<img width="1280" height="584" alt="image" src="https://github.com/user-attachments/assets/012bc156-72a5-412a-9e5e-624a4d90652d" />
